addons/project/static/src/views/project_collaborator_list/project_collaborator_list_view.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+import { listView } from '@web/views/list/list_view';
+import { InviteCollaboratorsListController } from './project_collaborator_list_controller';
+
+export const InviteCollaboratorsListView = {
+    ...listView,
+    Controller: InviteCollaboratorsListController,
+    buttonTemplate: 'collaborators.List_view.buttons',
+};
+
+registry.category('views').add('invite_collaborators', InviteCollaboratorsListView);
```

Comment:
```suggestion
registry.category('views').add('project_collaborators', ProjectCollaboratorsListView);
```
- In the future, we may want to add other functionalities to this class. Let's keep it as generic as possible :slightly_smiling_face: